apps/src/lab2/views/components/TextToSpeech.tsx

```diff
@@ -27,15 +33,19 @@ const TextToSpeech: React.FunctionComponent<TextToSpeechProps> = ({text}) => {
     speechSynthesis.speak(utterance);
   }, [text]);
 
+  if (!textToSpeechHasVoices) {
+    return null;
+  }
+
   return (
     <button
       className={moduleStyles.playButton}
       onClick={playText}
       type="button"
     >
       <FontAwesomeV6Icon
-        iconName={'play'}
-        iconStyle="solid"
+        iconName={'waveform-lines'}
```

Comment:
I do like waveform more than play. Wondering if a headphones or speaker icon might be clearer though? Looks like we use a speaker + play/pause icon in legacy labs

Reply:
Certainly open to trying other options. I feel like speakers are often used in reference to mute/volume control.  What I think could be good is a speaking mouth icon, but haven't found anything like that in our library so far.